libs/blocks/merch-card/merch-offer-select.js

```diff
@@ -3,12 +3,17 @@ import { decorateButtons } from '../../utils/decorate.js';
 import '../../deps/merch-offer-select.js';
 
 function createDynamicSlots(el, bodySlot) {
-  const price = createTag('h5', { class: 'merch-card-price' });
-  price.append(createTag('span', { slot: 'price', is: 'inline-price' }));
-  bodySlot.append(price);
-
+  const pricePlaceholder = el.querySelector("span[is='inline-price']");
+  if (pricePlaceholder) {
+    pricePlaceholder.parentNode.replaceChild(createTag('span', { slot: 'price', is: 'inline-price' }), pricePlaceholder);
```

Comment:
I don't understand why this is needed? creating a `span[is='inline-price']` without merch block seems wrong to me.